autonat.go

```diff
@@ -15,8 +15,11 @@ import (
 type NATStatus int
 
 const (
+	// NAT status is unknown
```

Comment:
maybe:

NATStatusUnknown implies that the status of the NAT has not been detected.

Reply:
yeah, that kind of deserves more explanation i guess.

Reply:
improved in https://github.com/libp2p/go-libp2p-autonat/pull/1/commits/7c097ed53dfd4e8539de4fa47dbc96ae9271be4c